make-promises-safe.js

```diff
@@ -3,11 +3,13 @@
 const event = 'unhandledRejection'
 
 process.on(event, function (err) {
-  console.error(err)
+  module.exports.logger.error(err)
```

Comment:
I would prefer us to just wrap this as a pure function, not as a `logger` object.

Reply:
so something like this?
```
module.exports.logError = function (err) { console.log(err); }
```

Reply:
I would go with `module.exports.logError = console.error`.